pycbc/psd/analytical_space.py

```diff
@@ -378,6 +379,7 @@ def averaged_lisa_fplus_sq_approx(f, len_arm=2.5e9):
     cwd = getcwd()
     if path.exists(cwd+"/AvFXp2_Raw.npy") is False:
         url = "https://zenodo.org/record/7497853/files/AvFXp2_Raw.npy"
+        ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
This seems a bit dangerous since you are changing this in the library. Every use would be affected. I think you need to use a way that only changes this for this case.  Also, why does this require no verification? Is that a bug on zenodo's end with a certificate? If so, would it resolve itself shortly?